### DISCUSSION TITLE
Please resolve and unpin stale incident notifications

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#49246] [Actions] [answered]
### Select Topic Area

General

### Body

@ettaboyle posted https://github.com/community/community/discussions/49104 pointing to https://www.githubstatus.com/incidents/sr7zptjd0b07 which is now resolved.

It'd be really helpful if when items like this are posted:
1. They're updated when the linked incident is resolved
2. They're unpinned 🚫 📌

**Comment by Akash1134** [comment, marked as answer]
Hi @jsoref , Thanks for highlighting this. I have updated the announcement and unpinned it from the category.

**Comment by ettaboyle** [comment]
Great call - thank you @jsoref 👍🏻
